js/src/redux/selectors/schemaBlocks.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * The schema validation results.
+ *
+ * @param {object} state The current state.
+ *
+ * @returns {object} The schema blocks validation results.
+ */
+export function getSchemaBlocksValidationResults( state ) {
```

Comment:
schemaBlocks in the name isn't neccessary, you're in the schema blocks selector. It returns BlockValidationResult[] so:
```suggestion
export function getBlockValidationResults( state ) {
```

Reply:
We lose this context when calling the selector, unfortunately:
```js
store.getSchemaBlocksValidationResults();
```
The selector is put directly on the store, and not in a `schemaBlocks` namespace.